Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SingleChoiceViewHolder.java

```diff
@@ -38,7 +38,34 @@ public void bind(SettingsItem item)
 
     if (item.getDescriptionId() > 0)
     {
-      mTextSettingDescription.setText(item.getDescriptionId());
+      // Special cases for SingleChoiceSetting options with long descriptions
```

Comment:
This doesn't really belong here. @mahdihijazi explained the issue with this approach in #7121: https://github.com/dolphin-emu/dolphin/pull/7121#discussion_r204168841

Reply:
I tried a different approach.